aecore_eqc/aec_sync_eqc.erl

```diff
@@ -107,7 +114,7 @@ is_blocked(Peer) ->
   aec_peers:is_blocked(pp(Peer)).
 
 is_blocked_post(S, [Peer], Res) ->
-  eq(Res, (Peer == error) orelse lists:member(full_peer(Peer), S#state.blocked)).
```

Comment:
full_peer is deleted everywhere... a bit surprising, but fine. But then the actual function is not deleted in this PR.

Reply:
Well spotted. Deleted in new commit.